packages/component-library/draft/Kaizen/SplitButton/styles.scss

```diff
@@ -166,7 +240,6 @@ $button-width: 124px;
 }
 
 .menuContainer {
-  position: absolute;
-  width: calc(#{$button-width} + 100%);
```

Comment:
Now we don't need this width calculation for the dropdown content making it to be the size of his parent.

Reply:
Sick! 👌